migration/identity-migration/src/main/java/io/camunda/migration/identity/handler/sm/RoleMigrationHandler.java

```diff
@@ -155,7 +155,16 @@ private Set<CreateAuthorizationRequest> createAuthorizationsForRoleAndPermission
     final Set<CreateAuthorizationRequest> authorizations =
         getAuthorizationsByAudience(audiences, permission, roleId, AuthorizationOwnerType.ROLE);
     if (migrationProperties.isResourceAuthorizationsEnabled()) {
-      authorizations.removeIf(auth -> !RBA_IRRELEVANT_RESOURCE_TYPES.contains(auth.resourceType()));
+      // Check if this is the zeebe-api:write permission
+      final String zeebeWritePermission = audiences.getZeebe() + ":write:*";
+      final boolean isZeebeWrite = permission.equals(zeebeWritePermission);
+
+      // If it's NOT zeebe-api:write, filter out RBA-relevant resource types
+      // When zeebe-api:write is present, keep all wildcard permissions even with RBA enabled
```

Comment:
It would be good to add the reason here why wildcard permissions are created for zeebe-api anyway. Otherwise the comment just explains what the code does, but that's something I can already see :)

Reply:
done